xccdf.xsd

```diff
@@ -1557,30 +1557,48 @@
         </xsd:complexContent>
     </xsd:complexType>
 
-    <xsd:complexType name="fixType" mixed="true">
+    <xsd:complexType name="fixType"> <!-- PROPOSED: REMOVED 'mixed="true"' -->
         <xsd:annotation>
             <xsd:documentation xml:lang="en"> Data type for the &lt;xccdf:fix&gt; element. The body
                 of this element contains a command string, script, or other system modification
                 statement that, if executed on the target system, can bring it into full, or at
                 least better, compliance with this &lt;xccdf:Rule&gt;. </xsd:documentation>
         </xsd:annotation>
-        <xsd:choice minOccurs="0" maxOccurs="unbounded">
-            <xsd:element name="sub" type="cdf:subType">
+        <!-- PROPOSED -->
+        <xsd:sequence>
+            <xsd:element name="fix-export" type="cdf:fixExportType" minOccurs="0" maxOccurs="unbounded">
```

Comment:
How is this different from using `<sub idref/>` Which is already supported in Tailoring and can already be used in the `<fix/> ` element for this?

Reply:
The `<sub idref>` functionality would be used within the `fix-content` element to allow values to be substituted in the text of the fix script/commands.  (text substitution)

The `fix-export` would be analogous to a `check-export` allowing XCCDF Values to be passed to the external content referenced by a `fix-content-ref`

Reply:
Cool. Can we add some of those words in the schema update to make it a little more clear?

Reply:
Starting down around line 1717, the `fixContentType` contains the descriptions of the `sub` element which indicates its usage of `Value` elements to be used for text substitution.  Is that good enough to suffice?  Or is there another place/more verbiage that could be used in that section?

```
 <xsd:element name="sub" type="cdf:subType">
    <xsd:annotation>
        <xsd:documentation xml:lang="en">Specifies an &lt;xccdf:Value&gt; or
            &lt;xccdf:plain-text&gt; element to be used for text substitution
        </xsd:documentation>
    </xsd:annotation>
</xsd:element>
```

Reply:
Sorry I wasn't clear. I was thinking about adding something like "The fix-export would be analogous to a check-export allowing XCCDF Values to be passed to the external content referenced by a fix-content-ref". But maybe that should be added to the doc and not the schema?

Reply:
Ah!  Yeah no problem.  I'll add that exact sentence.